packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -13,8 +13,13 @@ export class MatrixRenderer {
     public scaledColumnIndices = Float64Array.from([]);
     public scaledRowIndices = Float64Array.from([]);
     public columnIndicesPositions = Int32Array.from([]);
+    public colorIndices = Int32Array.from([]);
     public canvas!: OffscreenCanvas;
     public context!: OffscreenCanvasRenderingContext2D;
+    private colors: string[] = [];
+    private colorValues = Float64Array.from([]);
+    private readonly outsideRangeDieColor = 'rgba(218,223,236,1)';
```

Comment:
Where does this color come from? Should we have a token for it?

Reply:
This is the color used in the previous rendering for an empty die. It's light grey. Do tokens work in workers?

Reply:
No we can't leverage tokens from workers. Ideally we would want all the style related things (not just this color but the default color scale, the font face, etc) to all come from tokens. The tokens should be subscribed to from the element and observed for changes. The changes should then be passed to the worker, maybe as part of renderConfig, etc.

I wouldn't make it a requirement for this PR to avoid growing scope. but we should address both pieces in follow-ups:
1. Don't hard code style in the worker and instead observe tokens on the wafer and pass to the worker (higher priority architecture pattern)
2. Use / create tokens that are theme-aware for colors, etc. (lower priority theme integration / awareness)

Reply:
I updated https://github.com/ni/nimble/issues/2034 with the mentioned feedback